client/src/components/App.jsx

```diff
@@ -1,29 +1,63 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import axios from 'axios';
-import Review from './Review';
+import Review from './review';
 import Search from './Search';
 import Ratings from './Ratings';
 class App extends React.Component {
   constructor() {
     super();
     this.state = {
-    allReviews: [],
+      allReviews: [],
+      currentReviews: [],
+      currentSearchTerm: null,
     }
+    this.searchSubmit = this.searchSubmit.bind(this);
+    this.searchBarTextChange = this.searchBarTextChange.bind(this);
   }
+  
```

Comment:
Empty linebreaks: 
19 (Between contructor and componentDidMount methods)
36 (Below call to setState)